editor/index.js

```diff
@@ -46,6 +46,15 @@ window.jQuery( document ).on( 'heartbeat-tick', ( event, response ) => {
 	}
 } );
 
+const render = ( Component, props, target ) => {
```

Comment:
This file is starting to get muddied with various rendering implementations. I'd suggest:

- Create a DocBlock explaining the purpose of this function
- Maybe rename as `renderEditor`, so we can avoid import assignment of `elRender`, and assume the first argument is the `Editor` component (since it's not used any other way).